apidoc/Titanium/UI/WebView.yml

```diff
@@ -368,6 +463,69 @@ events:
       - name: url
         summary: The URL of the web document that is stopped.
 
+  - name: message
+    summary: Fired when a script message is received from a webpage.
+    description: |
+        This event get fired when you have added message handler using method 'addScriptMessageHandler'
+        and  webpage sends message to it.
+    platforms: [ipad, iphone]
+    since: "7.4.0"
+    properties:
+      - name: url
+        summary: URL of the web document being loaded.
+        type: String
+      - name: body
+        summary: The body of message sent from webview.
+        type: String
+      - name: name
+        summary: The name of the message handler to which the message is sent.
+        type: String
+      - name: isMainFrame
+        summary: |
+            A Boolean value indicating whether the frame is the web site's main frame or a subframe.
+        type: Boolean
+
+  - name: progress
+    summary: Fired when webpage download progresses.
+    description: |
+        This value ranges from 0.0 to 1.0 based on the total number of bytes expected to be received, 
+        including the main document and all of its potential subresources.
+    platforms: [ipad, iphone]
```

Comment:
Yes. On Android, we can detect page download progress via [WebChromeClient.onProgressChanged()](https://developer.android.com/reference/android/webkit/WebChromeClient.html#onProgressChanged(android.webkit.WebView,%20int)).

Would you also mind rephrasing the description please? How about to something like this?
> Provides a normalized value between 0.0 and 1.0 where 0.0 indicates nothing was downloaded and 1.0 means the page and its resources has finished downloading.

**Question:**
Does this value include iframes? I suspect it does not since an iframe can timeout and fail to download, but the main page can finish fine. I tried looking for info on the Android side about iframes, but it doesn't say anything either.